xrpl/models/base_model.py

```diff
@@ -73,6 +73,34 @@ def _value_to_json(value: XRPL_VALUE_TYPE) -> XRPL_VALUE_TYPE:
     return value
 
 
+def process_json_binary_codec_input(
```

Comment:
```suggestion
def process_xrpl_json(
```
The binary codec is irrelevant to the format of the JSON.